txpipe/twopoint.py

```diff
@@ -52,6 +52,7 @@ class TXTwoPoint(PipelineStage):
         'do_shear_pos': True,
         'do_pos_pos': True,
         'var_methods': 'jackknife',
+        'shape_noise': True,
```

Comment:
could you change the name of this to something that makes clear it's purpose, like `use_true_shear`?

Reply:
Done.